pkg/types/chain_test.go

```diff
@@ -42,7 +42,7 @@ func TestGetLinksPresent(t *testing.T) {
 
 	assert.Len(t, links, 2, "Expected 2 links")
 	assert.Equal(t, "Keplr", links[0].Name, "Expected Keplr link")
-	assert.Equal(t, "https://wallet.keplr.app/#/chain/governance?detailId=test", links[0].Href, "Wrong Keplr link")
+	assert.Equal(t, "https://wallet.keplr.app/chains/chain/proposals=test", links[0].Href, "Wrong Keplr link")
```

Comment:
I have a feeling that this would fail, as it's `proposals/test`, not `proposals=test`. Can you run `make test` locally to check if it's working? Somehow the CI checks won't start, not yet sure why.